it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/CheckAgencyControllerISpec.scala

```diff
@@ -693,13 +699,27 @@ trait CheckAgencyControllerISpec extends BaseISpec with SessionDataMissingSpec {
       val result = await(
         controller.invasiveTaxPayerOption(
           authenticatedAs(subscribingCleanAgentWithoutEnrolments)
-            .withFormUrlEncodedBody(("confirmResponse", "false"), ("confirmResponse-false-hidden-input", "40000000090000000"))
+            .withFormUrlEncodedBody(("variant", "utr"), ("utr", "40000000090000000"))
             .withSession("saAgentReferenceToCheck" -> "SA6012")))
 
       status(result) shouldBe 200
       bodyOf(result) should include(htmlEscapedMessage("error.utr.invalid.length"))
     }
 
+    "redirect to setUpIncomplete when successfully selecting ICannotProvideEitherOfTheseDetails" in {
+      givenUtrAGoodCombinationAndUserHasRelationshipInCesa("utr", "4000000009", "SA6012")
+
+      val result = await(
+        controller.invasiveTaxPayerOption(authenticatedAs(subscribingCleanAgentWithoutEnrolments)
+            .withFormUrlEncodedBody(("variant", "cannotProvide"))
+            .withSession("saAgentReferenceToCheck" -> "SA6012")))
+
+      status(result) shouldBe 303
+      redirectLocation(result) shouldBe Some(routes.StartController.setupIncomplete.url)
+
+      metricShouldExistsAndBeenUpdated("Count-Subscription-Could-Not-Provide-Tax-Payer-Identifier")
```

Comment:
The other metrics for the invasive checks start with `Count-Subscription-InvasiveCheck`, could we have the `-InvasaveCheck-` bit here as well?